src/app/core/modals/publish-modal/publish-modal.component.ts

```diff
@@ -116,9 +135,9 @@ export class PublishModalComponent extends DirectiveBase implements OnInit {
         });
 
         this.inputForm.statusChanges.filter(() => this.inputForm.valid).subscribe(() => {
-            const {name, project} = this.inputForm.getRawValue();
+            const {id, project} = this.inputForm.getRawValue();
             this.outputForm.patchValue({
-                appID: project + "/" + this.slugify.transform(name.toLowerCase()) + "/" + this.revision
+                appID: project + "/" + this.slugify.transform(id.toLowerCase()) + "/" + this.revision
```

Comment:
project + "/" + this.slugify.transform(name.toLowerCase()) + "/" + this.revision

[project, this.slugify.transform(name.toLowerCase()), this.revision].join("/");